### DISCUSSION TITLE
Solidus install error on ubuntu

### DISCUSSION THREAD

**Discussion by arulqb** [solidusio/solidus#4997] [Troubleshooting]
Running Rails from: /mnt/work/Shady/Work/Solidus/Project/solidus/sandbox
Traceback (most recent call last):
	21: from bin/rails:4:in `<main>'
	20: from bin/rails:4:in `require'
	19: from /home/arul/.gem/gems/railties-7.0.4.3/lib/rails/commands.rb:18:in `<top (required)>'
	18: from /home/arul/.gem/gems/railties-7.0.4.3/lib/rails/command.rb:48:in `invoke'
	17: from /home/arul/.gem/gems/railties-7.0.4.3/lib/rails/command/base.rb:87:in `perform'
	16: from /home/arul/.gem/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
	15: from /home/arul/.gem/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
	14: from /home/arul/.gem/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
	13: from /home/arul/.gem/gems/railties-7.0.4.3/lib/rails/commands/generate/generate_command.rb:21:in `perform'
	12: from /home/arul/.gem/gems/railties-7.0.4.3/lib/rails/command/actions.rb:14:in `require_application_and_environment!'
	11: from /home/arul/.gem/gems/railties-7.0.4.3/lib/rails/command/actions.rb:22:in `require_application!'
	10: from /home/arul/.gem/gems/railties-7.0.4.3/lib/rails/command/actions.rb:22:in `require'
	 9: from /mnt/work/Shady/Work/Solidus/Project/solidus/sandbox/config/application.rb:19:in `<top (required)>'
	 8: from /snap/ruby/284/lib/ruby/2.7.0/bundler.rb:174:in `require'
	 7: from /snap/ruby/284/lib/ruby/2.7.0/bundler/runtime.rb:58:in `require'
	 6: from /snap/ruby/284/lib/ruby/2.7.0/bundler/runtime.rb:58:in `each'
	 5: from /snap/ruby/284/lib/ruby/2.7.0/bundler/runtime.rb:69:in `block in require'
	 4: from /snap/ruby/284/lib/ruby/2.7.0/bundler/runtime.rb:69:in `each'
	 3: from /snap/ruby/284/lib/ruby/2.7.0/bundler/runtime.rb:74:in `block (2 levels) in require'
	 2: from /snap/ruby/284/lib/ruby/2.7.0/bundler/runtime.rb:74:in `require'
	 1: from /home/arul/.gem/gems/puma-5.6.5/lib/puma.rb:13:in `<top (required)>'
/home/arul/.gem/gems/puma-5.6.5/lib/puma.rb:13:in `require': /snap/core20/current/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /lib/x86_64-linux-gnu/libcrypto.so.3) - /home/arul/.gem/gems/puma-5.6.5/lib/puma/puma_http11.so (LoadError)

**Comment by kennyadsl** [comment]
I'm not familiar with this issue. Did you already try regenerating the sandbox with `bin/sandbox` from the root of the project?
